### PR TITLE
Partial item designation/undesignation/dispatch

### PR DESCRIPTION
**Pull request by Darshan-Chauhan**

### CODE REVIEW

File: app/controllers/items/partial_undesignate.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import Ember from "ember";
+import AjaxPromise from 'stock/utils/ajax-promise';
+const { getOwner } = Ember;
+
+export default Ember.Controller.extend({
+  promptUserPopUp: false,
+  orderPackage: null,
+  total: 0,
+  values: 0,
+  orderPackegesAndQuantities: null,
```
Comment (steveyken) [inline app/controllers/items/partial_undesignate.js]: minor typo: `orderPackagesAndQuantities`

**Comment by Darshan-Chauhan** [comment]
@steveyken , Yes as soon as Undispatch and StockIt side changes are done I'll add test cases for all scenarios. I think when this is done I'll be working only on test cases for couple of days. 😄

**Comment by steveyken** [comment]
@Darshan-Chauhan just wanted to check in and make sure js integration tests are still on the radar... this is one feature set that is worth having lots of good scenarios for.

**Comment by Darshan-Chauhan** [comment]
@steveyken , Yes I've added ticket for it (https://jira.crossroads.org.hk/browse/GCW-1410) and will start working on it.

**Pull request merged**